### DISCUSSION TITLE
Cloud "greyness" in daily predictions

### DISCUSSION THREAD

**Discussion by obones** [open-meteo/open-meteo#720] [Q&A]
Hello,

I'm using open-meteo data inside my home automation system which pushes weather representation to a nice panel.
On this one, I display icons for the forecast daily weather based on the Weather code which is fine, but I'm wondering if could get the cloud icons more precise by filling them with a gray level that goes from white cloud to very dark storm clouds.

Clearly, this is not something that is directly available, but I'm wondering if you would have suggestions as to how I could reconstruct this, either from daily forecast variables, or by aggregating hourly forecast variables.

Any suggestion is most welcome.

**Comment by patrick-zippenfenig** [comment]
Would the variable `cloud_cover` work? 100% is "only" overcast and not storm, but otherwise this is the simplest way to get a fractional cloud value.

**Reply by obones** [reply]
That would be fine indeed. 
Should I simply average the `cloud_cover` value over the 24 hours of a given day?

**Reply by patrick-zippenfenig** [reply]
Yes, you can average cloud cover for a day, or you could only consider hours between 6:00 and 18:00
